frontend/src/app/services/auth.service.ts

```diff
@@ -53,12 +49,17 @@ export class AuthService {
 
   private handleLoginSuccess(response: any) {
     const token = response.access_token;
+
+    if (!token) {
```

Comment:
Probably overly cautious, but could toss this in in case the response ever changes